tests/serialized_test_data_generation/check_data_dir.sh

```diff
@@ -44,23 +46,23 @@ if ! grep "WARNING: RUNNING WITH GT4PY_DEV ON" ${dir}/stdout.out &> /dev/null ;
 fi
 
 # make sure reasonable number of *.json files exists and they are all > 0 bytes
-if [ `/bin/ls ${dir}/*.json | wc -l` -lt 2 ] ; then
+if [ `find ${dir} -name '*.json' -print | wc -l` -lt 2 ] ; then
```

Comment:
Force of habit to use the same solution as in the rest of the code (I just did a search & replace of /bin/ls *). This one would not fail since there is a very limited number of *.json files, where as in the caes of * or *.dat the command line gets too long and the script breaks. Do you think it would make sense to leave this one as is?

Reply:
Oh awesome -- that's great, keep the change. I've been bitten by the * with too many files a lot here, so appreciate seeing this solution even if not needed for these files.